tests/characters/control/blockchain/test_web_control_blockchain.py

```diff
@@ -186,8 +189,25 @@ def test_bob_web_character_control_retrieve(bob_web_controller_test_client, retr
     response = bob_web_controller_test_client.post(endpoint, data=json.dumps({'bad': 'input'}))
     assert response.status_code == 400
 
-    del(params['alice_verifying_key'])
-    response = bob_web_controller_test_client.put(endpoint, data=json.dumps(params))
+    method_name, params = make_retrieve_control_request().values()
+    tmap = b64encode(bytes(enacted_blockchain_policy.treasure_map)).decode()
+    params['treasure_map'] = tmap
+    response = bob_web_controller_test_client.post(endpoint, data=json.dumps(params))
+    assert response.status_code == 200
+    assert response_data['result']['cleartexts']
+
+    method_name, params = make_retrieve_control_request().values()
+    wrong_tmap = TreasureMap(m=0)
+    wrong_tmap.prepare_for_publication(
+            blockchain_bob.public_keys(DecryptingPower),
+            blockchain_bob.public_keys(SigningPower),
+            blockchain_alice.stamp,
+            b'Wrong!')
+    wrong_tmap = b64encode(bytes(wrong_tmap)).decode()
+    params['treasure_map'] = wrong_tmap
+    with pytest.raises(Exception):
```

Comment:
Is there a more specific `Exception` that can be checked here? Or maybe an error response code is more appropriate - I guess this is the question in the comment below...

Reply:
Yes, we should catch a more specific error here.

Reply:
Let me see if I can make a treasure map with 1 Ursula instead